### PR TITLE
[CHANGE] Bradley tow limit/ T-72 no HEAT-FS

### PR DESCRIPTION
**Pull request by JessevGool**

### CODE REVIEW

**Comment by JessevGool** [comment]
Removed extra bradley tow magazines, made sure ammo gets added after 2nd removal

File: cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf [hunk 1]
```diff
@@ -307,6 +308,12 @@ JST_fnc_vehRespawn =
 	{
 		_unitVar removeMagazinesTurret [_x, [0]];
 	} forEach VehBannedMagazines;
+	// Add custom magazines
+	{
+		_unitVar addMagazineTurret _x
+	} forEach ReplaceMagazines;
+	// Save loadout for ace rearm
+	_unitVar setVariable ["ace_rearm_scriptedLoadout", true, true];
```
Comment (tanaaKa) [inline cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf:316]: Oh shit. This was supposed to be here. It must've gotten overwritten in a merge somewhere.

Comment (tanaaKa) [reply cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf:316]: No, it's definitely here https://github.com/Totyano/Eternal-Babylon-TVT/blob/033ac0532ebca2794402e81141cc70094cbbafb7/cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf#L389

Comment (JessevGool) [reply cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf:316]: Might be able to go without the one on 316 but it def needs the addMagazineTurret there

Comment (JessevGool) [reply cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf:316]: @tanaaKa just tested without the 2nd ace_rearm and it needs to be there
The code at the bottom only gets ran at mission start, not when a vehicle respawns

Comment (tanaaKa) [reply cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf:316]: Yeah, I figured, but if we run a `setVariable` at start after the magazines change, that should save the loadout regardless.

**Review by Totyano — APPROVED** [review]

**Pull request merged**